### PR TITLE
CMCL-646: Collider smoothing time not working

### PR DESCRIPTION
**Pull request by gaborkb**
### Purpose of this PR

https://jira.unity3d.com/browse/CMCL-646
Regression, smoothing time stopped working.

I could not find a difference between damping behaviours.

### Testing status

- [ ] Added an automated test
- [x] Passed all automated tests
- [x] Manually tested 

### Documentation status

- [ ] Updated [CHANGELOG](https://keepachangelog.com/en/1.0.0/)
- [ ] Updated README (if applicable)
- [ ] Commented all public classes, properties, and methods
- [ ] Updated user documentation

### CODE REVIEW

**Comment by gaborkb** [comment]
Fixing tests.

File: Tests/Runtime/CmColliderTests.cs [hunk 1]
```diff
@@ -44,56 +52,73 @@ public IEnumerator CheckSmoothingTime()
...
+            yield return null;
+            yield return null;
             yield return null;
             Assert.That(originalCamPosition == m_Vcam.State.FinalPosition, Is.False);
             
             obstacle.transform.position = originalCamPosition + Vector3.left * 100f; // move obstacle out of the way
 
+            yield return null;
+            yield return null;
+            yield return null;
```
Comment (glabute) [inline Tests/Runtime/CmColliderTests.cs:66]: All these yield return null statements, here and elsewhere in this file, look sketchy.  Why are they here?

File: Tests/Runtime/CmColliderTests.cs [hunk 2]
```diff
@@ -44,56 +52,73 @@ public IEnumerator CheckSmoothingTime()
...
             yield return null;
             Assert.That(pos1 == m_Vcam.State.FinalPosition, Is.False);
+            
+            obstacle.transform.position = originalCamPosition + Vector3.left * 100f; // move obstacle out of the way
+
+            yield return null;
+            yield return null;
+            yield return null;
+            Assert.That(originalCamPosition == m_Vcam.State.FinalPosition, Is.True);
```
Comment (glabute) [inline Tests/Runtime/CmColliderTests.cs:98]: Do you really want hard equality here? (and elsewhere in this file)

Comment (gaborkb) [reply Tests/Runtime/CmColliderTests.cs:87]: Yes, I expect exact equality. Though, I could add some tolerance. I ran the tests hundreds of times, and it was ok with exact equality.

**Review by glabute — APPROVED** [review]

**Pull request merged**